docs/data-sources/ccm_certificates.md

```diff
@@ -0,0 +1,101 @@
+---
+subcategory: "Cloud Certificate Manager (CCM)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_ccm_certificates"
+description: |-
+  Use this data source to get the list of CCM SSL certificates.
+---
+
+# huaweicloud_ccm_certificates
+
+Use this data source to get the list of CCM SSL certificates.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_ccm_certificates" "test" {
+  status = "ALL"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the certificate name.
+
+* `status` - (Optional, String) Specifies the certificate status.  
+  The options are as follows:
+  + **ALL**: All certificate status.
+  + **PAID**: The certificate has been paid and needs to be applied for from the CA.
+  + **ISSUED**: The certificate has been issued.
+  + **CHECKING**: The certificate application is being reviewed.
+  + **CANCELCHECKING**: The certificate application cancellation is being reviewed.
+  + **UNPASSED**: The certificate application fails.
+  + **EXPIRED**: The certificate has expired.
+  + **REVOKING**: The certificate revocation application is being reviewed.
+  + **REVOKED**: The certificate has been revoked.
+  + **UPLOAD**: The certificate is being managed.
+  + **CHECKING_ORG**: The organization verification is to be completed.
+  + **ISSUING**: The certificate is to be issued.
+  + **SUPPLEMENTCHECKING**: Additional domain names to be added for a multi-domain certificate are being reviewed.
+  
+  Defaults to **ALL**.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID to which the CCM certificate belongs.
```

Comment:
Specifies the enterprise project ID to which the CCM certificates belong.

Reply:
OK